zulipterminal/model.py

```diff
@@ -182,6 +183,8 @@ def __init__(self, controller: Any) -> None:
 
         self._subscribe_to_streams(self.initial_data["subscriptions"])
 
+        self._alert_words = self.initial_data.get("alert_words", None)
```

Comment:
While using `get` is sort of helpful here, in case it's not in the data, this would cause issues later if it were to be None. We don't validate structure but rely on the expected data being present, for the most part.

In this case, mypy doesn't complain since initial_data is a `Dict[str, Any]`, and it doesn't care what happens with the `Any` 👎.

This means we need to give the type for `self._alert_words`, and only from that point onwards is it guaranteed to be non-None, so a `.get` here makes little sense, unless we're going to do a separate assert before this - or simply let a KeyError drop out in case of an error.

Reply:
Upon looking at the `self._alert_words = self.initial_data.get("alert_words", None)` part, I saw that when the user has no alert_words, it returns an empty list(I was probably thinking `None` earlier,not sure). So changed it to `self._alert_words: List[str] = self.initial_data["alert_words"]`.